### ISSUE TITLE
hwloc_get_obj_below_by_type() failing, task/affinity plugin may be required

### ISSUE THREAD

**Issue by gqqnbig** [gqqnbig/shine-cluster#56]
error log:

> slurmstepd-eureka: error: task/cgroup: hwloc_get_obj_below_by_type() failing, task/affinity plugin may be required to address bug fixed in HWLOC version 1.11.5
> slurmstepd-eureka: error: task/cgroup: task[0] unable to set taskset '0x0'
> slurmstepd-eureka: error: *** JOB 5788 ON eureka CANCELLED AT 2021-09-12T13:00:18 ***

When commiting a job, sometimes slurm gives this error. It happened once back in April when Lu Wang reported this. Now I encountered again on eureka.

**Comment by gqqnbig** [comment]
duplicate of #29

**Comment by gqqnbig** [comment]
Check the version of mpi-servers on eureka

**Comment by wulamao** [comment]
![image](https://user-images.githubusercontent.com/8916829/142579167-565f3b46-cf9d-4d15-9116-79a787686ed3.png)

It is confirmed that the currently running version is equipped with the fix.

**Comment by gqqnbig** [comment]
Looks like Eureka has the potential fix but the error still occured. 

keep on watching.

Do not merge PR until we make sure the commit indeed fixes the bug.

**Comment by gqqnbig** [comment]
Revisit on Feb 19, 2022.
